poc_iot_verifier/migrations/6_add_entropy_start_time.sql

```diff
@@ -0,0 +1 @@
+alter table poc_report add entropy_start_time timestamptz default now() not null;
```

Comment:
does this need any sort of explicit rollback step or can it be inferred completely from the migration?

Reply:
Hmm, good q. Probably best to add rollback I suppose. But it did work as is in the live deployment we tried.

Reply:
[1a47c12](https://github.com/helium/oracles/pull/182/commits/1a47c1266c0e3ceac9214472bf4f9744b245a734) Can confirm that this works locally with sqlx-cli